### ISSUE TITLE
add  preset settings and 1 more enhancement

### ISSUE THREAD

**Issue by wolfkingal2000** [Lymphatus/caesium-image-compressor#298]
I suggest adding a feature to the software that allows users to quickly add and change preset settings, so they don't have to adjust the settings manually every time. Additionally, displaying the size of all files (similar to how the size of a photo is shown) would enhance the software's functionality and improve the user experience.
![Caesium_Image_Compressor_uUL3odet80](https://github.com/user-attachments/assets/f6c1a823-66a9-45b5-aa63-abaeca164eb4)
show me like top photo 
![S1n4ZffiT2](https://github.com/user-attachments/assets/08f7cada-0cff-4143-ba6e-79b46316656f)

**Comment by Lymphatus** [comment]
> Additionally, displaying the size of all files (similar to how the size of a photo is shown) would enhance the software's functionality and improve the user experience.

Sorry but I didn't get what are you suggesting. Can you please explain a little bit further? Thanks!

**Comment by wolfkingal2000** [comment]
> > Additionally, displaying the size of all files (similar to how the size of a photo is shown) would enhance the software's functionality and improve the user experience.
> 
> Sorry but I didn't get what are you suggesting. Can you please explain a little bit further? Thanks!

sure The first one is obvious, and the second one, like a photo that shows the size reduction later, I want the overall size of the photos together to show their change, how much of the overall size of the photos has been reduced.

**Comment by Lymphatus** [comment]
Ok so if I understood correctly, you'd like to see how many bytes have been reduced?
For example if a photo went from 2.5Mb to 1.7Mb you'd like to see that saved `0.8Mb`? Is that correct?

**Comment by wolfkingal2000** [comment]
> Ok so if I understood correctly, you'd like to see how many bytes have been reduced?
> For example if a photo went from 2.5Mb to 1.7Mb you'd like to see that saved `0.8Mb`? Is that correct?

Yes in second photo i send i show purple rectangle show  me how much saved from all photos

**Comment by Lymphatus** [comment]
Actually the number at the bottom shows the total size of the imported files and not how much you saved, but I think I understood your request
